siteapp/forms.py

```diff
@@ -27,13 +27,6 @@ class Meta:
         model = Portfolio
         fields = ['title', 'description']
 
-    def clean(self):
-        """Extend clean to validate portfolio name is not reused."""
```

Comment:
True, the issue I had was trying to update the description while keeping the title the same. This validation prevents the update from happening. Is there a way to avoid using clean for this particular use-case?

Reply:
I would think it should continue to block that case, so that people don't rename a Portfolio to have the same name as another. Maybe all it needs is to keep this validation and add a message when the validation fails, such as "Name already used. Cannot save Portfolio with this name. Please try another." and it could redirect back to the Portfolio edit page.

Reply:
Adding a updateportfolio hidden input to check for in the clean method

Reply:
But the same "clean" validation of the name should happen on both create and edit, correct? This new logic seems to skip this validation if the portfolio name is being edited. I think we still need to check for name duplicates on edit, and just inform the user if the name is invalid (on both edit and create).